Source/RapyutaSimulationPlugins/Private/Robots/Turtlebot3/TurtlebotBurger.cpp

```diff
@@ -27,9 +27,13 @@ void ATurtlebotBurger::SetupBody()
     WheelRight = CreateDefaultSubobject<UStaticMeshComponent>(TEXT("WheelRight"));
     CasterBack = CreateDefaultSubobject<UStaticMeshComponent>(TEXT("CasterBack"));
 
-    if (RootComponent)
+    // NOTE: This is critical, otherwise GetTransform() always returns one of [DefaultRoot] as initial one at spawning due to its being just a non-physics scene component

```

Comment:
- Do you mean this is a issue at the spawning time only?
- If this is a issue to general robot, shouldn't be part of base robot?

Reply:
Yeah I added `SetBaseMeshComp()` for RRBaseRobot, so child classes, if creating a custom base mesh comp, could use to automatically make it new RootComp.